src/signals/incident/components/form/MapSelectors/Caterpillar/context/context.tsx

```diff
@@ -13,7 +13,6 @@ export const initialValue: SelectValue = {
     featureTypes: [],
     wfsFilter: '',
     icons: [],
-    legendItems: [],
```

Comment:
Is the file `src/signals/incident/components/form/MapSelectors/Caterpillar/context/context.tsx` still used? The AssetSelect context is referenced in other places. Can't this file be removed?

Reply:
removed it